pkg/services/ngalert/notifier/receivers.go

```diff
@@ -200,7 +200,8 @@ func (am *Alertmanager) TestReceivers(ctx context.Context, c apimodels.TestRecei
 func newTestAlert(c apimodels.TestReceiversConfigParams, startsAt, updatedAt time.Time) types.Alert {
 	var (
 		defaultAnnotations = model.LabelSet{
-			"summary": "Notification test",
+			"summary":          "Notification test",
+			"__value_string__": "metric=500",
```

Comment:
Perhaps we should add the real formatting, WDYT?
```suggestion
			"__value_string__": "[ metric='foo' labels={instance=bar} value=10 ]",
```

Reply:
`__value_string__` is a private label that is removed from notifications, do we want to add it to the test alert?

Reply:
In this case we only use it to populate the test notification with a value. I think it is removed before we send the notification, so it shouldn't show up in the payload. I'll doublecheck.

Reply:
Then what is the purpose of `__value__string__` here? Is it so it can be used for template expansion in a template?

Reply:
Yes, exactly.